packages/data-table/src/helpers/configHelpers.ts

```diff
@@ -0,0 +1,28 @@
+import { cloneDeep } from 'lodash'
+import { ChartConfig } from '../types/ChartConfig'
+
+/* editConfigKeys
```

Comment:
I feel like it's much easier to use Object.assign or the spread operator.

{....config, filters: []}

{...config, table: {...config.table, show: true}

if you really need to set something deeply nested lodash already has this function:

_.set(obj, ['x', 'y', 'z'], 'hello');

https://lodash.com/docs/4.17.15#set